tools/rdock/rdock.xml

```diff
@@ -0,0 +1,76 @@
+<tool id="rdock" name="rDock" version="1.0">
+    <description>Binding Mode Prediction in Proteins/RNA</description>
+    <requirements>
+        <requirement type="package" version="2013.1">rDock</requirement>
+    </requirements>
+    <command>
+        <![CDATA[ 
+    	ln -s $inputmol2 inputmol2.mol2 && cat $inputprm | sed "s|RECEPTOR_FILE.*|RECEPTOR_FILE inputmol2.mol2|g" | sed "s|REF_MOL.*|REF_MOL ligand.sd|g" > new.prm; 
+        cp -r $__tool_directory__/tools/rdock/data .;

```

Comment:
I don't think this is going to work, `$__tool_directory__` is translated to the directory where the Galaxy tool XML file is located.